SpatialGDK/Source/SpatialGDK/Public/SpatialView/CommandResponse.h

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Templates/UniquePtr.h"
+#include <improbable/c_worker.h>
+#include <improbable/c_schema.h>
+
+namespace SpatialGDK
+{
+
+struct CommandResponseDeleter
+{
+	void operator()(Schema_CommandResponse* CommandResponse) const noexcept
+	{
+		if (CommandResponse)
+		{
+			Schema_DestroyCommandResponse(CommandResponse);
+		}
+	}
+};
+
+using OwningCommandResponsePtr = TUniquePtr<Schema_CommandResponse, CommandResponseDeleter>;
+
+// An RAII wrapper for component data.
+class CommandResponse
+{
+public:
+	// Creates a new component data.
+	explicit CommandResponse(Worker_ComponentId ComponentId, Worker_CommandIndex CommandIndex);
```

Comment:
Do we need explicit here?

Reply:
Use explicit everywhere you don't have a reason not to.

Reply:
When is the reason to not have it?